app/build.gradle.kts

```diff
@@ -51,13 +51,12 @@ android {
     }
     kotlinOptions {
         jvmTarget = "1.8"
-        freeCompilerArgs += "-opt-in=androidx.compose.material3.ExperimentalMaterial3Api"
```

Comment:
Is there a reason it changes to `@OptIn`, instead of `freeCompilerArgs` `-opt-in`?

Other than explicit versus global preferences?